relayer-cli/src/commands/create/channel.rs

```diff
@@ -1 +1,151 @@
+use abscissa_core::{Command, Options, Runnable};
 
+use ibc::ics02_client::state::ClientState;
+use ibc::ics03_connection::connection::IdentifiedConnectionEnd;
+use ibc::ics04_channel::channel::Order;
+use ibc::ics24_host::identifier::{ChainId, ConnectionId, PortId};
+use ibc::Height;
+use ibc_relayer::channel::Channel;
+use ibc_relayer::config::StoreConfig;
+use ibc_relayer::connection::Connection;
+use ibc_relayer::foreign_client::ForeignClient;
+
+use crate::commands::cli_utils::{spawn_chain_runtime, ChainHandlePair, SpawnOptions};
+use crate::conclude::{exit_with_unrecoverable_error, Output};
+use crate::prelude::*;
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct CreateChannelCommand {
```

Comment:
In other commands, we use lowercase sentences in help messages.

Reply:
Nice catch, thank you. 62b0a26e